cli/src/commands/util.rs

```diff
@@ -62,7 +62,17 @@ pub(crate) struct UtilCompletionArgs {
 
 /// Run backend-dependent garbage collection.
 #[derive(clap::Args, Clone, Debug)]
-pub(crate) struct UtilGcArgs {}
+pub(crate) struct UtilGcArgs {
+    /// Time threshold
+    ///
+    /// By default, only objects and operations older than 2 weeks are pruned.
+    ///
+    /// Only the string "now" can be passed to this parameter. Support for
+    /// arbitrary absolute and relative timestamps will come in a subsequent
+    /// release.
+    #[arg(long, verbatim_doc_comment)]
+    prune: Option<String>,
```

Comment:
Perhaps call it something like `--expire-all` rather than have the "now" syntax?

It's probably fine to keep the "now" syntax if you're planning to immediately implement support for other values, but I think it'd be fine to postpone that. We can then avoid the awkward syntax until it's useful.

Reply:
Yeah, "prune" in GC is a bit strange, although I would say that the strangeness it that GC sometimes doesn't prune, not the name of the parameter itself (since it really does control whether things are pruned or not). Even if we say that users will never want to not prune, we still want to allow users to control the time threshold (presumably), and users could still set the time threshold to "infinite" (with the effect of not pruning anything), so naming this "prune" might still make sense.

I think "expire" has the same meaning as "prune", so I would prefer to stick to the Git name, but can switch if people want.

To @ilyagr 's point, we are planning to implement support for other values, but not right now (so, postponing it). I think it's more awkward to have `--expire-all` and then `--expire` as opposed to having `--expire=now` then `--expire=anything`, though.

Reply:
The current behavior of "--prune" in Git was introduced in 58e9d9d472 (gc: make --prune useful again by accepting an optional parameter, 2009-02-14), if people want to look at it (probably only contains things that people already know, though).

Reply:
I prefer `--expire` than `--prune` because `--expire` implies the argument is some kind of time. `--prune` sounds like a flag to turn on some feature of gc.

Reply:
I agree. It sounds like we will not prune eligible objects unless you pass `--prune`.

Reply:
OK, let's go with "expire".